libraries/consul_config.rb

```diff
@@ -79,6 +80,7 @@ class ConsulConfig < Chef::Resource
       attribute(:rejoin_after_leave, equal_to: [true, false], default: true)
       attribute(:server, equal_to: [true, false], default: true)
       attribute(:server_name, kind_of: String)
+      attribute(:session_ttl_min, kind_of: String)      
```

Comment:
I got the diff of the commit and applied it to my repo, then fixed the whitespace and committed.  I can't get that to this PR, right?  @matelukas needs to do that, right?